src/components/Calculator.vue

```diff
@@ -1,168 +1,320 @@
+                    this.second_argument += !this.second_argument ? '0' : '';
+                    this.second_argument += this.isValueExist(this.second_argument, regExpForDot) != -1 ? '' : '.';
+                }
+
+                this.result();
+            },
+
+            onOperationClick(operation) {
+
+                if (!this.first_argument) {
+                    this.first_argument = '0';
+                }
+
+                this.first_argument = this.deleteDotIfNoDigitAfter(this.first_argument)
+
+                if (this.second_argument) {
+                    this.second_argument = this.deleteDotIfNoDigitAfter(this.second_argument)
+                    this.onGetResult();
+                }
+
+                this.operation = operation;
+                this.result();
+            },
+
+            deleteDotIfNoDigitAfter(argument) {
+
+                if (this.isValueExist(argument, /\./) == (argument.length - 1)) {
+                    return argument.slice(0, argument.length - 1)
+                } else {
+                    return argument
+                }
+            },
+
+            onGetResult() {
+
+                try {
+                    if (!this.first_argument || (!this.first_argument && !this.second_argument)) {
+                        return;
+                    }
+
+                    if (!this.operation && !this.lastOperation) {
+                        return;
+                    }
+
+                    if (this.first_argument && !this.operation && !this.second_argument && this.lastOperation) {
+                        this.operation = this.lastOperation;
+                        this.second_argument = this.lastSecond_argument;
+                    }
+
+                    if (this.first_argument && this.operation && !this.second_argument) {
+                        this.second_argument = this.first_argument;
+                    }
+
+                    if (this.second_argument === '0' && this.operation === '/') {
+                        throw new Error('Can not divide by zero');
+                    }
+
+                    if (!this.second_argument && this.operation) {
+                        this.second_argument += this.first_argument;
+                    }
+
+                    if (this.second_argument) {
+                        this.second_argument = this.deleteDotIfNoDigitAfter(this.second_argument)
+                    }
 
-    .engineer-block {
-      display: inline-grid;
-      grid-template-columns: repeat(3, 1fr);
-      grid-template-rows: repeat(5, 45px);
-      grid-template-areas: 'large_button large_button . ';
-      grid-gap: $grid-gap-wide-screen;
-      width: 40%;
-      box-sizing: border-box;
-      margin-top: 10px;
-      padding-right: $grid-gap-wide-screen;
-
-      @media screen and (max-width: 620px) {
-        display: none;
-      }
+                    this.result();
+
+                    const result = eval(this.calc_screen).toString();
```

Comment:
here you use `eval` to make calculations. so you are not limited by only two numbers and operation symbol between them. As I see what you need - is string with expression. You can use, for example, array, and push there operators and numbers, then join it into string. If you want to add validation, it will be easy to take previous value and choose following behavior (as you do with dot and dividing).
Not sure about huge validation, (maybe it was the task), but usually calculator says, that invalid format is inserted and you may restrict input simple operator right after another simple operator.

Reply:
yes, this can be done differently. But in  the description of the project was only about expressions with one mathematical operation